views/layouts/dashboard.handlebars

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>THE NODOMON-CHOICE</title>
```

Comment:
Please change this from THE NODOMON-CHOICE to just Nodomon

Same with the links.